sites/public/src/components/home/HomeRegions.tsx

```diff
@@ -11,14 +11,28 @@ export const HomeRegions = () => {
     enum: region,
   }))
 
+  const regionImageUrls: Map<RegionEnum, string> = new Map([
```

Comment:
question: We can talk about this in a group setting. I would prefer if we would decouple custom changes specifically for Detroit outside of the release PRs. Ideally the release PRs would be just a direct pull over of the commits in core. And if there aren't any merge conflicts it won't require much testing of the PR because everything has already been tested in core. Then if custom changes are needed a separate PR is created with just those so we have an isolated PR that should be tested.